paperqa/readers.py

```diff
@@ -76,6 +76,12 @@ def parse_pdf(path: Path, doc: Doc, chunk_chars: int, overlap: int) -> List[Text
 def parse_txt(
     path: Path, doc: Doc, chunk_chars: int, overlap: int, html: bool = False
 ) -> List[Text]:
+    """Parse a document into chunks, based on tiktoken encoding.
```

Comment:
might want to add the hyperlink to tiktoken for reference here in the docstring
https://github.com/openai/tiktoken

Reply:
done